generators/app/templates/ext-command-js/vsc-extension-quickstart.md

```diff
@@ -30,7 +30,7 @@
 * Open the debug viewlet (`Ctrl+Shift+D` or `Cmd+Shift+D` on Mac) and from the launch configuration dropdown pick `Extension Tests`.
```

Comment:
- When trying it out, I don't see the `Extensions Tests` launch configuration. Should I still get it? I have the recommended extensions installed.
- A point `Make sure the recommended Extension Test Runner extension is installed` could be helpful
- Also could mention that the test can  ow also be run from the test viewlet

Reply:
Thanks, done!